pom.xml

```diff
@@ -129,6 +129,7 @@
                     <artifactId>license-maven-plugin</artifactId>
                     <version>4.1</version>
                     <configuration>
+                        <!--suppress MavenModelInspection -->
```

Comment:
Stop Intellij complaining that it can't resolve the property as it can't tell that its defined by a plug-in.